apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -265,17 +319,70 @@ class ReviewTab extends Component {
     );
   }
 
+  renderPeerDropdown(reviewablePeers, onSelectPeer) {
+    return (
+      <PeerSelectDropdown
+        text={javalabMsg.reviewClassmateProject()}
+        peers={reviewablePeers}
+        onSelectPeer={onSelectPeer}
+      />
+    );
+  }
+
+  renderBackToMyProject(onClickBackToProject) {
+    return (
+      <Button
+        text={javalabMsg.returnToMyProject()}
+        color={Button.ButtonColor.white}
+        icon={'caret-left'}
+        size={Button.ButtonSize.default}
+        iconStyle={styles.backToProjectIcon}
+        onClick={onClickBackToProject}
+        style={styles.backToProjectButton}
+      />
+    );
+  }
+
   render() {
-    const {comments, forceRecreateEditorKey, isReadyForReview} = this.state;
+    const {viewAsCodeReviewer, viewAs} = this.props;
+
+    const {
+      comments,
+      forceRecreateEditorKey,
+      isReadyForReview,
+      errorSavingReviewableProject,
+      errorLoadingReviewblePeers,
+      reviewablePeers,
+      projectOwnerName
+    } = this.state;
 
     return (
       <div style={styles.reviewsContainer}>
         <div style={styles.reviewHeader}>
+          {viewAs !== ViewType.Teacher &&
+            !errorLoadingReviewblePeers &&
+            (viewAsCodeReviewer
+              ? this.renderBackToMyProject(this.onClickBackToProject)
+              : this.renderPeerDropdown(reviewablePeers, this.onSelectPeer))}
```

Comment:
What does this look like if there are no projects ready for review?

Reply:
I believe just an empty dropdown - I can double check if that's the intended experience.